components/layout_2020/positioned.rs

```diff
@@ -175,20 +190,22 @@ impl PositioningContext {
     pub(crate) fn adjust_static_position_of_hoisted_fragments(
         &mut self,
         parent_fragment: &Fragment,
+        index: PositioningContextLength,
```

Comment:
Nit: The comment above says

> This function is used to update that static position at every level of the fragment tree

But this can now be limited with `index`, so the comment may need some update.

Reply:
Okay. I've updated this comment.